src/web/pages.rkt

```diff
@@ -134,19 +147,16 @@
   ;   vars: array of n string variable names
   ;   points: array of size m like [[x0, x1, ..., xn], ...] where x0 etc. 
   ;     are ordinals representing the real input values
-  ;   error: object with fields {start, target, end}, where each field holds 
-  ;     an array like [y0, ..., ym] where y0 etc are bits of error for the output 
-  ;     on each point
+  ;   error: JSON dictionary where keys are {start, end, target1, ..., targetn}.
```

Comment:
I mean, this is ok, but isn’t it kind of ugly? Why not make one entry, target, point to an array?

Reply:
My idea as to generalize the `report.js` as much as possible. This made it one for-loop instead of multiple. And multiple start and end seemed of less probability. I could revert it, but it was to make the JS much less easier to read